kevm-pyk/src/kevm_pyk/foundry.py

```diff
@@ -391,79 +392,37 @@ def foundry_prove(
     setup_methods: dict[str, str] = {}
     contracts = unique({test.split('.')[0] for test in tests})
     for contract_name in contracts:
+        if 'setUp' in foundry.contracts[contract_name].method_by_name:
+            setup_methods[contract_name] = f'{contract_name}.setUp'
+
+    def _init_apr_proof(_init_problems: tuple[str, str]) -> APRProof | APRBMCProof:
```

Comment:
Consider renaming the argument `_init_problems` -> `_init_problem`, since it represents a single pair of contract name and contract method.

Reply:
Done.